Lombiq.HelpfulExtensions/Assets/Vendors/Trumbowyg/plugins/highlight/trumbowyg.highlight.js

```diff
@@ -0,0 +1,323 @@
+            'ocaml': 'OCaml',
+            'opencl': 'OpenCL',
+            'parigp': 'PARI/GP',
+            'objectpascal': 'Object Pascal',
+            'pcaxis': 'PC-Axis',
+            'px': 'PC-Axis',
+            'peoplecode': 'PeopleCode',
+            'pcode': 'PeopleCode',
+            'php': 'PHP',
+            'phpdoc': 'PHPDoc',
+            'php-extras': 'PHP Extras',
+            'plsql': 'PL/SQL',
+            'powerquery': 'PowerQuery',
+            'pq': 'PowerQuery',
+            'mscript': 'PowerQuery',
+            'powershell': 'PowerShell',
+            'properties': '.properties',
+            'protobuf': 'Protocol Buffers',
+            'py': 'Python',
+            'q': 'Q (kdb+ database)',
+            'qml': 'QML',
+            'rkt': 'Racket',
+            'jsx': 'React JSX',
+            'tsx': 'React TSX',
+            'renpy': 'Ren\'py',
+            'rest': 'reST (reStructuredText)',
+            'robotframework': 'Robot Framework',
+            'robot': 'Robot Framework',
+            'rb': 'Ruby',
+            'sas': 'SAS',
+            'sass': 'Sass (Sass)',
+            'scss': 'Sass (Scss)',
+            'shell-session': 'Shell session',
+            'solidity': 'Solidity (Ethereum)',
+            'solution-file': 'Solution file',
+            'sln': 'Solution file',
+            'soy': 'Soy (Closure Template)',
+            'sparql': 'SPARQL',
+            'rq': 'SPARQL',
+            'splunk-spl': 'Splunk SPL',
+            'sqf': 'SQF: Status Quo Function (Arma 3)',
+            'sql': 'SQL',
+            'tap': 'TAP',
+            'toml': 'TOML',
+            'tt2': 'Template Toolkit 2',
+            'trig': 'TriG',
+            'ts': 'TypeScript',
+            't4-cs': 'T4 Text Templates (C#)',
+            't4': 'T4 Text Templates (C#)',
+            't4-vb': 'T4 Text Templates (VB)',
+            't4-templating': 'T4 templating',
+            'uscript': 'UnrealScript',
+            'uc': 'UnrealScript',
+            'vbnet': 'VB.Net',
+            'vhdl': 'VHDL',
+            'vim': 'vim',
+            'visual-basic': 'Visual Basic',
+            'vb': 'Visual Basic',
+            'wasm': 'WebAssembly',
+            'wiki': 'Wiki markup',
+            'xeoracube': 'XeoraCube',
+            'xojo': 'Xojo (REALbasic)',
+            'xquery': 'XQuery',
+            'yaml': 'YAML',
+            'yml': 'YAML'
+        }
+    };
+
+    function highlightIt(text, language, lineHighlight) {
+        return [
+            '<pre class="language-' + language + '" ' + (lineHighlight ? 'data-line="' + lineHighlight + '"' : '') + '>',
+            '<code class="language-' + language + '">' + Prism.highlight(text, Prism.languages[language]) + '</code>',
+            '</pre>',
+        ].join('');
+    }
+
+    function escapeHtml(html) {
+        return $('<div/>').text(html).html();
+    }
+
+    function buildHighlightOptions(trumbowyg) {
+        var languageNames = trumbowyg.o.plugins.highlight.languageNames;
```

Comment:
`var` shouldn't be used ever since ES6 came out. Use `const` or `let`

Reply:
This is a vendor (3rd party) js file, we don't maintain those.